### ISSUE TITLE
Fix CI

### ISSUE THREAD

**Issue by dbrgn** [ccc-ch/ccc-ch.ch#117]
I'd suggest moving to CircleCI. I've made good experiences there.

(Maybe something I can do during 34c3)

**Comment by dbrgn** [comment]
I FINALLY found out why Travis is still being triggered – it was listed in the required status checks in the branch protection :)

I'll try to set up CircleCI now.

**Comment by dbrgn** [comment]
CircleCI is set up, we need to fix the new warning though: https://github.com/ccc-ch/ccc-ch.ch/pull/135#issuecomment-392356486

**Comment by gnurbs** [comment]
I think this warning is warranted - it's introduced by my addition of the english subsite

**Comment by dbrgn** [comment]
Yep, with "fix" I meant "handle", whatever that means :slightly_smiling_face:

**Comment by dbrgn** [comment]
While the locale problems are fixed, we still need to handle the missing translation warnings somehow:

https://circleci.com/gh/ccc-ch/ccc-ch.ch/54

**Comment by dbrgn** [comment]
Moved to Gitlab.
